OrbitBase/Logging.cpp

```diff
@@ -4,12 +4,14 @@
 
 #include <OrbitBase/Logging.h>
 
-static absl::Mutex log_file_mutex;
+static absl::Mutex log_file_mutex(absl::kConstInit);
 std::ofstream log_file;
 
 void InitLogFile(const std::string& path) {
   absl::MutexLock lock(&log_file_mutex);
-  CHECK(!log_file.is_open());
+  if (log_file.is_open()) {
+    PLATFORM_ABORT();
```

Comment:
Why don't we want the message from `CHECK` here?

Reply:
this happens under the lock - CHECK will invoke LogToFile which will fail because the mutex is not recursive here. Since this method is part of Logging system initialization it shouldn't call logging.

Reply:
added comment explaining this.